<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DlgHidden</class>
 <widget class="QWidget" name="DlgHidden">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>560</width>
    <height>399</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Manage</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="sizeConstraint">
      <enum>QLayout::SetMinimumSize</enum>
     </property>
     <property name="leftMargin">
      <number>12</number>
     </property>
     <property name="rightMargin">
      <number>12</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btnSelect">
       <property name="toolTip">
        <string>Selects all tracks in the table below.</string>
       </property>
       <property name="text">
        <string>Select All</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnPurge">
       <property name="toolTip">
        <string>Purge selected tracks from the library.</string>
       </property>
       <property name="text">
        <string>Purge</string>
       </property>
       <property name="checkable">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnUnhide">
       <property name="toolTip">
        <string>Unhide selected tracks from the library.</string>
       </property>
       <property name="text">
        <string>Unhide</string>
       </property>
       <property name="shortcut">
        <string>Ctrl+S</string>
       </property>
       <property name="checkable">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTableView" name="m_pTrackTablePlaceholder">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="showGrid">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../res/mixxx.qrc"/>
 </resources>
 <connections/>
</ui>
